src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,30 @@
+import { render, screen } from "@testing-library/react";
+import SearchResults from "../conponents/SearchResults";
+
+describe("SearchResults", () => {
+  const validProps = {
+    searchResults: ["https://unsplash.com/photos/uIf6H1or1nE", "https://unsplash.com/photos/wCujVcf0JDw"]
```

Comment:
You do not need to use actual links to images in your tests, you can just pass them strings e.g.
```
searchResults: ["image1", "image2"]
```
then you could test that those strings appear in your code, e.g.
```
expect(screen.getAllByRole("img")[0]).toHaveAttribute("src", validProps.searchResults[0])
```